controller/src/main/java/com/softserve/borda/controllers/TicketController.java

```diff
@@ -280,11 +283,11 @@
         }
     }
 
-    @DeleteMapping("/tickets/{ticketId}/deleteMember/{userId}")
+    @DeleteMapping("/tickets/{ticketId}/deleteMember")
```

Comment:
Bad resource naming.
Please read https://restfulapi.net/resource-naming/ doc and change naming.

Reply:
Fixed in FixResourcesPaths branch/ticket, already merged in main